### ISSUE TITLE
EPERM: operation not permitted removing a directory

### ISSUE THREAD

**Issue by Danita** [gpbl/webpack-cleanup-plugin#5]
Hi! I get this error:

```
  return binding.unlink(pathModule._makeLong(path));
                 ^

Error: EPERM: operation not permitted, unlink 'G:\Projects\Panel\app\dist\fonts'
    at Error (native)
    at fs.unlinkSync (fs.js:980:18)
    at Array.forEach (native)
    at Compiler.<anonymous> (G:\Projects\Panel\app\node_modules\webpack-cleanup-plugin\lib\WebpackCleanupPlugin.js:47:15)
```

During installation npm warned me that `fsevents` was not compatible with my system. Could be this dependency?

**Comment by gpbl** [comment]
:wave: Danita, 

looking at [this question](http://stackoverflow.com/questions/8496212/node-js-fs-unlink-function-causes-eperm-error) on stack overflow, it does not seem related to windows. I think your `G:\Projects\Panel\app\dist\fonts` directory stills contains files and the plugin is trying to delete it. I don't know why this still happens, as it should have been fixed with https://github.com/gpbl/webpack-cleanup-plugin/pull/3. 

Going to check this soon and write here back.

Thanks for the bug report!

**Comment by gpbl** [comment]
It should be fixed with 0.2.0! please check it out and reopen the issue if it doesn't still work. Thanks!

**Comment by Danita** [comment]
Great, thanks! :smile:
